EIPS/eip-7512.md

```diff
@@ -18,71 +18,89 @@ The proposal aims to create a standard for an onchain representation of audit re
 There are four parties:
 
 - User: The end user that wants to bridge their token
-- Bridge Operator: The operator is maintaining the bridge
-- Bridge: The contract the user will interact with to trigger the bridging
+- Bridge Operator: The operator that maintains the bridge
+- Bridge: The contract the user will interact with to trigger the bridge operation
 - Validator: The contract that validates that a token can be bridged
 
-As a first (1) step the bridge operator should define the keys/accounts for the auditors from which audits are accepted for the token registration process. 
+As a first (1) step, the bridge operator should define the keys/accounts for the auditors from which audits are accepted for the token registration process. 
 
-With this the user (or token owner) can trigger the registration flow (2). There are two steps (3 and 6) that will be performed: verify that the provided audit is valid and has been signed by a trusted auditor (4) and check that the token contract implements [ERC-1155](./eip-1155.md) (7).
+With this, the user (or token owner) can trigger the registration flow (2). There are two steps (3 and 6) that will be performed: verify that the provided audit is valid and has been signed by a trusted auditor (4), and check that the token contract implements the bridge's supported token standard ([ERC-1155](./eip-1155.md)) (7).
 
-With the checks being done it is still recommended to have some manual check in place by the operator to activate a token for bridging (10). This step could be omitted if there is a strong trust in the auditor or if an ERC provides strong compatibility guarantees. 
+After the audit and token standard validations have been performed, it is still advisable to have some form of manual intervention in place by the operator to activate a token for bridging (10). <!-- This step could be omitted if there is strong trust in the auditor or if an ERC provides strong compatibility guarantees. -->
 
-Once the token is available on the bridge the users can start using it (11). 
+Once the token has been activated on the bridge, Users can start bridging it (11). 
 
 ## Specification
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
 ### Audit Properties
 
 - Auditor
-    - `name` - Name of the auditor (i.e. for displaying to the user)
-    - `uri` - Uri to retrieve more information about the auditor
-    - `authors` - A list of authors that contributed to this audit. This SHOULD be the persons that audited the contracts and created the audit.
+    - `name`: Name of the auditor (i.e. for displaying to the user)
```

Comment:
Hmm I thought I did this change before as it was recommended [here](https://github.com/ethereum/EIPs/pull/7652#discussion_r1318707244) :see_no_evil: